src/index.js

```diff
@@ -3,6 +3,8 @@ import Editor from './editor';
 import { isElement } from 'underscore';
 import polyfills from 'utils/polyfills';
 import PluginManager from './plugin_manager';
+import 'utils/touchpolyfill.js';
+import 'utils/DragDropTouch.js';
 
```

Comment:
I'm quite reluctant to merge those 2 files inside the core, we don't need them. Who wants to enable the editor with touch devices probably has already polyfills for them.

Reply:
Since GrapesJs is making use of the native DragDrop events, the single options was : 

- Rewrite all DrapDrop functionalities making use of standard multiplatform API's .
- Polyfill these events for platform not supporting DragDrop natively .

I was new to GrapesJS, and the poly fill option was the main proposal in the feature request thread (without any objection). So it was the best option.

If you can help me to add it in a modular way , we can look it .